Libraries/NativeAnimation/RCTNativeAnimatedModule.m

```diff
@@ -22,6 +27,8 @@ @implementation RCTNativeAnimatedModule
   NSMutableArray<AnimatedOperation> *_preOperations;
 }
 
+@synthesize bridge = _bridge;
```

Comment:
Just serious, is this a special reason to use `syntesize` here? Why should not use just a ivar?

Reply:
Or why don't add this as a prop to class interface inside *this* (.m) file?
To me, `@syntesize` always looks like or something redundant/old-fasioned or something very specific.

Reply:
From what I  understand @synthesize is used to implement properties from a protocol without having to redefine them (https://github.com/facebook/react-native/blob/master/React/Base/RCTBridgeModule.h#L87). It's pretty much used for every native module that needs to access the bridge.